Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM openjdk:11-slim
+LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+
+RUN apt-get update && \
+    apt-get install -y curl && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV TZ=Europe/London
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" > /etc/timezone
+
+RUN addgroup --gid 2000 --system appgroup && \
+    adduser --uid 2000 --system appuser --gid 2000
+
+# Install AWS RDS Root cert into Java truststore
+RUN mkdir /home/appuser/.postgresql \
+  && curl https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem \
+    > /home/appuser/.postgresql/root.crt
+RUN curl https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem \
```

Comment:
hopefully you won't need the 2015 cert - we're going to start deleting them all next week as prod will have been updated everywhere